sdk/tcp_listener.go

```diff
@@ -14,6 +16,14 @@ const (
 )
 
 func newTCPListener(address string, pluginName string) (net.Listener, string, error) {
+	if address == "" {
+		return nil, "", fmt.Errorf("plugin address cannot be empty")
+	}
+	addrParts := strings.SplitN(address, ":", 2)
```

Comment:
I verified, with a test program, that `net.SplitHostPort` successfully splits empty hostnames on Windows, both IPv4 and IPv6. As far as I can tell, the [function](https://golang.org/src/net/ipsock.go?s=3288:3354#L99) does not have Windows-specific code. It does not do much validation: basic IPv6 checks, and exactly one `:` (outside of IPv6 `[]`, if present).